src/posix/platform/netif.cpp

```diff
@@ -2250,7 +2250,7 @@ void platformNetifSetUp(void)
 #if OPENTHREAD_CONFIG_REFERENCE_DEVICE_ENABLE
     otIcmp6SetEchoMode(gInstance, OT_ICMP6_ECHO_HANDLER_ALL);
 #else
-    otIcmp6SetEchoMode(gInstance, OT_ICMP6_ECHO_HANDLER_DISABLED);
+    otIcmp6SetEchoMode(gInstance, OT_ICMP6_ECHO_HANDLER_RLOC_ALOC_ONLY);
```

Comment:
Does linux need this change? I think this is only required for NCP build. Probably set this in `src/ncp/ncp_base_mtd`.

Reply:
linux doesn't need `OT_ICMP6_ECHO_HANDLER_RLOC_ALOC_ONLY`, moved this to `ncp_base`, could you take another look?